### DISCUSSION TITLE
Dados do equipamento na O.S

### DISCUSSION THREAD

**Discussion by ghost** [RamonSilva20/mapos#1733] [Ideas]
Quero deixar a minha sugestão aqui que no momento do cadastro da O.S. é preciso ter os dados do equipamento que está dando entrada com os seguintes dados:
Data e hora da entrada, marca, modelo e número de série, além de informar os dados dos acessórios que acompanham ele.

**Comment by FlaviusR** [comment]
Existe uma tabela de 'equipamentos', so ainda nao consegui implementá-la de modo correto

**Comment by borgesthiago** [comment]
Acho bastante importante inclusive no **cadastro do Cliente** permitir cadastrar os equipamentos dele e na OS selecionar qual está com defeito.

**Comment by henri0387** [comment]
Fala galera, acho que como dito acima e interessante ter o numero de serie do equipamento. mas o mesmo pode ser colocado na observação. atualmente utilizo esse campo com esse intuito. 
talvez isso pode ser contornado com um campo pra por um imagem da etiqueta, afinal uma imagem fala mais que mil palavras.

**Comment by candidocassio** [comment]
Heyy, ótima sugestão. Estou tentando implantar o software aqui no meu trabalho, fazemos manutenções em equipamentos do poder público, um campo assim seria muito util para uma organização como essa, pois o patrimonio que identica cada equipamento é unico. Assim poderiamos gerar relatórios e saber o histórico de manutenção de cada máquina.

**Comment by WagnerFFreitas** [comment]
Bom dia a todos, Saberia dizer se a tabela equipamento foi implantada ou tem alguma atualizacao?

**Comment by andremp15** [comment]
Olá pessoal não só o cad
![IMG-20221230-WA0029](https://user-images.githubusercontent.com/117945999/210120708-950a9515-bb92-450d-8de7-8a066723ccdd.jpg)
astro do equipamento mas da emissão da os era bom que saísse duas vias na mesma folha A4 vou postar o modelo aqui para todo mundo ver eu fiz no Corel draw
